capgains/commands/capgains_calc.py

```diff
@@ -64,6 +64,13 @@ def _filter_calculated_transaction(transaction, year):
     return True
 
 
+def _get_total_gains(calculated_dicts):
+    total = 0
+    for calculated_dict in calculated_dicts:
+        total += calculated_dict['capital_gain']
+    return "{0:,.2f}".format(total)

```

Comment:
Do the string formatting outside of this method. This method should just return the total as a float value.

Reply:
Fixed